src/terminal/terminal_handlers.c

```diff
@@ -1422,7 +1431,7 @@ int guac_terminal_ctrl_func(guac_terminal* term, unsigned char c) {
     switch (c) {
 
         /* Alignment test (fill screen with E's) */
-        case '8':
+        case 'GUAC_TERMINAL_ALIGNMENT_TEST':
```

Comment:
This will not compile. Presumably you meant:

    case GUAC_TERMINAL_ALIGNMENT_TEST:

and

    #define GUAC_TERMINAL_ALIGNMENT_TEST '8'